Kernel/VM/MemoryManager.cpp

```diff
@@ -78,8 +85,10 @@ void MemoryManager::initialize_paging()
     // 8 MB   -> MAX            Userspace physical pages (available for allocation!)
 
     // Basic virtual memory map:
-    // 0 MB   -> 8MB            Identity mapped.
-    // 0xc0000000-0xffffffff    Kernel-only virtual address space.
+    // 0x00000000-0x00100000    Identity mapped for Kernel Physical pages
```

Comment:
What does Kernel Physical mean?

Reply:
Physical in this case are the pages handed out by `allocate_supervisor_page` for things such as DMA/HW devices. I'll quantify that comment a bit more.